diode-proto/diode/v1/ingester.proto

```diff
@@ -14,7 +14,10 @@ option go_package = "github.com/netboxlabs/diode-sdk-go/diode/v1/diodepb";
 // An ingest entity wrapper
 
 message Entity {
-  google.protobuf.Timestamp timestamp = 1 [(validate.rules).timestamp.required = true, (validate.rules).timestamp.lt_now = true];
+  google.protobuf.Timestamp timestamp = 1 [
```

Comment:
interestingly, it looks like buf reformatted this when generating? possibly preferable?

Reply:
maybe need a way to do this update so it always does this ...

Reply:
for readability 👍